samples/api-event-source/java/template.yaml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Description: |
+  An example RESTful service.
+
+Resources:
+  ExampleFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: java8
+      Handler: com.amazonaws.examples.Handler::handleRequest
+      CodeUri: ./target/APIEventSource-1.0.jar
+      Events:
+        TestEndpoints:
+          Type: Api
+          Properties:
+            Path: /
+            Method: any
+  LambdaInvokePermission:
```

Comment:
you don't need this. SAM will automatically add it for you

Reply:
But I do need it if I want to run the example in the Lambda console, right? That's why I added it. Otherwise it'll run locally but not after I upload it. I'm fine removing it, though. Please advise.